staging/vhost-device-gpu/README.md

```diff
@@ -13,10 +13,53 @@ A virtio-gpu device using the vhost-user protocol.
 ## Options
 
 ```text
-  -s, --socket-path <SOCKET>  vhost-user Unix domain socket
-  -g, --gpu-mode <GPU_MODE>   [possible values: virgl-renderer, gfxstream]
-  -h, --help                  Print help
-  -V, --version               Print version
+  -s, --socket-path <SOCKET>
+          vhost-user Unix domain socket
+
+  -g, --gpu-mode <GPU_MODE>
+          The mode specifies which backend implementation to use
+          
+          [possible values: virgl-renderer, gfxstream]
+
+  -c, --capset <CAPSET>
+          Comma separated list of enabled capsets
+
+          Possible values:
+          - virgl:            [virglrenderer] OpenGL implementation, superseded by Virgl2
+          - virgl2:           [virglrenderer] OpenGL implementation
+          - venus:            [virglrenderer] Vulkan implementation
```

Comment:
the ResourceCreate2D and other gpu commands fails with `Rutabaga component error` when the `venus` capset is set, have you tried it with the resource_blob PR? maybe it needs it to function properly?